src/main/webapp/app/lecture/lecture.component.html

```diff
@@ -134,6 +138,7 @@ <h2 id="page-heading">
                             <td>{{ lecture.visibleDate | artemisDate }}</td>
                             <td>{{ lecture.startDate | artemisDate }}</td>
                             <td>{{ lecture.endDate | artemisDate }}</td>
+                            <td>{{ lecture.ingested?.toString() }}</td>
```

Comment:
```suggestion
                            <td>{{ lecture.ingested }}</td>
```
Is it actually necessary?